conversion/nifti_write.py

```diff
@@ -54,17 +54,21 @@ def nifti_write(inImg, prefix= None):
     TIME_UNITS = 0
 
     SPACE2RAS = _space2ras(hdr['space'])
-    xfrm_nhdr= np.matrix(np.vstack((np.hstack((hdr['space directions'][:3,:3].T,
-                        np.reshape(hdr['space origin'][~np.isnan(hdr['space origin'])],(3,1)))),[0,0,0,1])))
-    xfrm_nifti= SPACE2RAS @ xfrm_nhdr
```

Comment:
`space origin` does not have a `none` in the specification, so removing it.
http://teem.sourceforge.net/nrrd/format.html